saleor/order/migrations/0181_order_subtotal_as_a_field.py

```diff
@@ -0,0 +1,25 @@
+from django.apps import apps as registry
+from django.db import migrations
+from django.db.models.signals import post_migrate
+
+from .tasks.saleor3_19 import update_order_subtotals
+
+
+def trigger_update_order_subtotals(apps, _schema_editor):
+    def on_migrations_complete(sender=None, **kwargs):
+        update_order_subtotals.delay()
```

Comment:
For main, the celery task should be converted to normal migration. The celery task is only valid for the migration that is placed in released version

Reply:
done